src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -221,6 +224,7 @@ private static <E extends Enum<E>> List<Object> getEnumPossibleValues(Class<E> e
         new Parameter(DEBUG_DIR_PARAM_NAME, ParameterType.STRING, "Directory to dump debug files", LfNetworkParameters.DEBUG_DIR_DEFAULT_VALUE, Collections.emptyList(), ParameterScope.TECHNICAL),
         new Parameter(INCREMENTAL_TRANSFORMER_VOLTAGE_CONTROL_OUTER_LOOP_MAX_TAP_SHIFT_PARAM_NAME, ParameterType.INTEGER, "Incremental transformer voltage control maximum tap shift per outer loop", IncrementalTransformerVoltageControlOuterLoop.DEFAULT_MAX_TAP_SHIFT),
         new Parameter(SECONDARY_VOLTAGE_CONTROL_PARAM_NAME, ParameterType.BOOLEAN, "Secondary voltage control simulation", LfNetworkParameters.SECONDARY_VOLTAGE_CONTROL_DEFAULT_VALUE),
+        new Parameter(SECONDARY_VOLTAGE_CONTROL_PILOT_POINT_VOLTAGE_SENSI_EPSILON_PARAM_NAME, ParameterType.DOUBLE, "Secondary voltage control, control unit voltage to pilot point voltage minimal sensitivity", SecondaryVoltageControlOuterLoop.SENSI_V_V_EPS_DEFAULT_VALUE),
```

Comment:
The name of the parameter is net clear enough. Pilot point is enough for me to understand that we are in the secondary voltage control outer loop. Maybe:
`CONTROLLER_TO_PILOT_POINT_VOLTAGE_SENSI_EPSILON` could be better.